face_rec_python:
  confidence_level: 25
